SPARKCONF.md

```diff
@@ -0,0 +1,155 @@
+##Spark Configuration
+
+oni-ml main component uses Spark and Spark SQL to analyze network events and produce a list of least probable events
+or most suspicious. 
+
+Before running oni-ml users need to configure the component. Here are our recommended settings.  
+
+### Yarn tuning
+
+oni-ml Spark application has been developed and tested on CDH [Yarn](http://spark.apache.org/docs/latest/running-on-yarn.html) 
+clusters. Careful tuning of the Yarn cluster may be necessary before analyzing large amounts of data with oni-ml.
+
+For small data sets, perhaps a couple of hundreds of gigabytes of CSV data, default _Yarn_ configurations should be enough
+but if users try to analyze hundreds of gigabytes of data in parquet format it's pr/obable that it don't work; _Yarn_ most likely will start killing
+ containers or will terminate the application with Out Of Memory errors.
+
+This document is not intended to explain in detail how to tune _Yarn_ but instead it provides some articles we used for this task:
+
+- [Cloudera Tuning Yarn](https://www.cloudera.com/documentation/enterprise/5-6-x/topics/cdh_ig_yarn_tuning.html)
+- [Tune Hadoop Cluster to get Maximum Performance](http://crazyadmins.com/tag/tuning-yarn-to-get-maximum-performance)
+- [Best Practices for YARN Resource Management](https://www.mapr.com/blog/best-practices-yarn-resource-management)
+
+Users need to keep in mind that to get a Spark application running, especially for big data sets, it might take more than
+one try before getting any results.
+
+### Spark Properties
```

Comment:
suggested title: "Configuring Spot's Usage of Spark"

I'd emphasize that this section is about configuring Spot/ONI to use Spark in a certain way, not the configuration of Spark itself